### PR TITLE
ENG-331 Component usage endpoints

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/content/ContentTypeResourceController.java [hunk 1]
```diff
@@ -54,6 +54,7 @@
 
 @RestController
 public class ContentTypeResourceController implements ContentTypeResource {
+    public static final String COMPONENT_ID = "contentTypes";
```
Comment (Kerruba) [inline plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/content/ContentTypeResourceController.java:57]: Would it actually make sense to have this singular?

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
All is Well

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by Kerruba** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
This build is still running, I think two concurrent builds were triggered

**Comment by entando-service** [comment]
All is Well

**Pull request merged**